### ISSUE TITLE
Error Data:  503

### ISSUE THREAD

**Issue by markpdev** [gtklocker/bittrex-orderbook#3]
Today I started to get the following error:

Error Message:  Negotiate Unknown
Exception:  undefined
Error Data:  503

Even the demo code provided throws the same error.

const BittrexOrderBook = require('bittrex-orderbook');
const bit = new BittrexOrderBook;

bit.market('BTC-XMR').on('bidUpdate', (market) => {
    console.log('XMR bids', market.bids.top(5));
});
bit.market('BTC-ETH').on('askUpdate', (market) => {
    console.log('ETH asks', market.asks.top(5));
});

**Comment by gtklocker** [comment]
Thanks for reporting this, I was able to successfully reproduce it.

Looks like it has something to do with Bittrex using Cloudflare now. I'll try to investigate.

**Comment by gtklocker** [comment]
An update:

The API will be taken off Cloudflare soon, according to Bittrex staff, so this shouldn't be a problem for much longer. While trying to mitigate the Cloudflare protection though, I noticed `signalr-client` didn't work even when I passed in a correct Cloudflare cookie, while reproducing the entire negotiation sequence using `curl` worked just fine. My availability isn't very high lately so I can't dive in and find the root issue there anytime soon.
